base/src/main/java/com/austindroids/austinfeedsme/data/eventbrite/EventbriteDataSource.kt

```diff
@@ -5,26 +5,24 @@ import com.austindroids.austinfeedsme.data.Event
 import com.austindroids.austinfeedsme.data.EventsDataSource
 import com.austindroids.austinfeedsme.data.EventsRepository
 import io.reactivex.Single
-import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.functions.Consumer
+import io.reactivex.functions.Function
 import io.reactivex.schedulers.Schedulers
 import okhttp3.OkHttpClient
 import retrofit2.Retrofit
 import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
 import retrofit2.converter.gson.GsonConverterFactory
 import timber.log.Timber
-import java.util.*
-import kotlin.collections.ArrayList
 
 
 /**
  * Created by darrankelinske on 8/26/16.
  */
 class EventbriteDataSource(val eventsRepository: EventsRepository) : EventsDataSource {
 
+    private val observableList = ArrayList<Single<EventbriteEvents>>()
```

Comment:
Hmm, I don't trust this class.

Reply:
it is unwieldy for sure

Reply:
Reading the whole class, it might work just fine, though.